NStackSDK/NStackSDK/Classes/Translations/TranslationManager.swift

```diff
@@ -96,6 +96,15 @@ public class TranslationManager {
         self.logger.customName = "-Translations"
     }
 
+    public func translationString(keyPath: String) -> String? {
+        if keyPath.characters.count == 0 {
+            return nil;
+        }
+        
+        let langDict = self.translationsDictionary as NSDictionary
+        return langDict.value(forKeyPath: keyPath) as? String
```

Comment:
This whole thing can be done as `return translationsDictionary.value(forKeyPath: keyPath) as? String` :P